docs/ossm/ossm2-migration/README.md

```diff
@@ -8,22 +8,117 @@ Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to
-    - Grafana: `spec.addons.grafana.enabled=false`
-    - Tracing: `spec.tracing.type=None`. See [here](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/distributed-tracing-and-service-mesh#ossm-distr-tracing-assembly) for instructions on how to configure OpenShift Service Mesh 3.0 with OpenShift Distributed Tracing as a replacement for the tracing addon.
+  - Disabled addons. The OpenShift Service Mesh 3.0 operator does not manage addons. If these are enabled on your `SerivceMeshControlPlane`, you will now need to manage these yourself by first migrating to an external instance and then disabling the addon.
+    - Prometheus: `spec.addons.prometheus.enabled=false`. See [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-integrating-with-user-workload-monitoring_observability) for instructions on how to configure your `ServiceMeshControlPlane` with OpenShift Monitoring as a replacement for the prometheus addon. Note that these instructions also detail how to install a standalone `Kiali` resource and it is ok to do both of these at the same time. If you are not using OpenShift Monitoring and are using a different prometheus solution, you may follow [these](https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/service_mesh/service-mesh-2-x#integration-with-external-prometheus-installation) instructions instead.
+    - Tracing: `spec.tracing.type=None`. See [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-configuring-distr-tracing-tempo_observability) for instructions on how to configure your `ServiceMeshControlPlane` with OpenShift Distributed Tracing as a replacement for the tracing addon. If not using OpenShift Distributed Tracing, you can add the custom tracing endpoint to your `spec.meshConfig.extensionProviders` configuration and create the corresponding `Telemetry` resource.
+    - Kiali: `spec.addons.kiali.enabled=false`. You will need to create a standalone `Kiali` resource to replace the addon. If you haven't already done this as part of the previous steps for Prometheus or Tracing, see [here](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/kiali-operator-provided-by-red-hat#ossm-install-kiali-operator_ossm-kiali-assembly) for instructions on how to install and configure Kiali as a standalone resource.
+    - Grafana: `spec.addons.grafana.enabled=false`. Grafana is not supported in OpenShift Service Mesh 3.0.
   - IOR is disabled.
```

Comment:
I believe we have documentation or both disabling IOR and migrating off of gateways that we could link to here:
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration

Reply:
Perfect! I'll add those.